### ISSUE TITLE
[patch] need signal.h on Fedora 14

### ISSUE THREAD

**Issue by GoogleCodeExporter** [ashish21590/redis#365]
```
What steps will reproduce the problem?
1. make test

What is the expected output? What do you see instead?

After completing the direct redis tests, the Makefile attempts to build and 
test hiredis. The test harness fails to compile:

[...]
cc -c -std=c99 -pedantic -O2 -fPIC -Wall -W -Wwrite-strings    -g -ggdb   test.c
test.c: In function ‘main’:
test.c:375:5: warning: implicit declaration of function ‘signal’
test.c:375:12: error: ‘SIGPIPE’ undeclared (first use in this function)
test.c:375:12: note: each undeclared identifier is reported only once for each 
function it appears in
test.c:375:21: error: ‘SIG_IGN’ undeclared (first use in this function)
make: *** [test.o] Error 1


What version of the product are you using? On what operating system?

2.2-alpha4-8-g645e996 built from github source on Fedora 14 i686.


Please provide any additional information below.

The attached patch resolves the issue; 'make test' recurses to deps/hiredis, 
which passes the first 5 tests.
```

Original issue reported on code.google.com by `gi...@xiph.org` on 5 Nov 2010 at 4:04

Attachments:
- [0001-Include-signal.h-for-signal-and-associated-constants.patch](https://storage.googleapis.com/google-code-attachments/redis/issue-365/comment-0/0001-Include-signal.h-for-signal-and-associated-constants.patch)

**Comment by GoogleCodeExporter** [comment]
```
Thanks for the patch! Just as in issue #365, very weird this didn't surface 
earlier.
```

Original comment by `pcnoordh...@gmail.com` on 5 Nov 2010 at 4:15
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Just as in issue #364 of course...
```

Original comment by `pcnoordh...@gmail.com` on 5 Nov 2010 at 4:17

**Comment by GoogleCodeExporter** [comment]
```
This is now fixed on master.
```

Original comment by `pcnoordh...@gmail.com` on 5 Nov 2010 at 4:31
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
This got fixed under issue #364. Thanks!
```

Original comment by `gi...@xiph.org` on 5 Nov 2010 at 4:36
